articles/search/search-more-like-this.md

```diff
@@ -22,22 +22,22 @@ ms.date: 11/04/2019
 
 By default, the contents of all top-level searchable fields are considered. If you want to specify particular fields instead, you can use the `searchFields` parameter. 
 
-You cannot use moreLikeThis on searchable sub-fields in a [complex type](search-howto-complex-data-types.md).
+You cannot use `MoreLikeThis` on searchable sub-fields in a [complex type](search-howto-complex-data-types.md).
 
 ## Examples
 
-All examples below uses [hotels sample from Quickstart](search-get-started-portal.md).
+All examples below use [hotels sample from Quickstart](search-get-started-portal.md).
```

Comment:
Missing article "the": "below use" -> "below use the"

Reply:
Done :) Thank you!